CMSgov/onemac
# OY2 933 design forms

Endpoint:
https://d1r53fiywnu1bu.cloudfront.net/

Story link: 
https://qmacbis.atlassian.net/browse/OY2-933?atlOrigin=eyJpIjoiYjBlODY1MTMzMTAyNDQ2Njg5ZmNjZjYyZjljNTc3NDUiLCJwIjoiaiJ9

Prototype link:
https://www.figma.com/file/NYWgJIpXMsyOOrZm0ceQHk/MACSTACK-DESIGNS?node-id=3313%3A4148

Changes:
- Updated index.scss and every form page to get the form styling as close to the Figma prototype as possible.
- Added the PageTitleBar as a header element.
- Updated FAQ and Dashboard minimally to add the PageTitle Bar and adjust styling where needed.
- Please note, the nav alignment is not correct - will be fixed later

Test:
1. Login as a Developer because Okta will not work in this branch
a. Please note, the developer login button and page are not necessarily styled as they should.
3. Clear cache if needed
4. Click each one of the form links from the dashboard and check
a. That the form looks “right”
b. That the link “what’s my” goes the the right section of the FAQ
c. That the form works as expected
d. That the Submit button is small
9. Submit the form
10. Select Form submission from dashboard and check that the form looks right

https://d1r53fiywnu1bu.cloudfront.net/
